src/lib/dnssd/tests/TestTxtFields.cpp

```diff
@@ -174,8 +174,8 @@ void TestGetDeviceType(nlTestSuite * inSuite, void * inContext)
     strcpy(dt, "1234");
     NL_TEST_ASSERT(inSuite, GetDeviceType(GetSpan(dt)) == 1234);
 
-    // overflow a uint16
-    sprintf(dt, "%" PRIu32, static_cast<uint32_t>(std::numeric_limits<uint16_t>::max()) + 1);
+    // overflow a uint32
+    sprintf(dt, "%" PRIu32, static_cast<uint32_t>(std::numeric_limits<uint32_t>::max()) + 1);
```

Comment:
This is changing the test. It used to put a large number into the string and then test how it gets parsed.  But now it's just putting in 0, no?

To have an equivalent of what we used to have it would need to be something like:
```suggestion
    sprintf(dt, "%" PRIu64, static_cast<uint64_t>(std::numeric_limits<uint32_t>::max()) + 1);
```